docs/usage/PRODUCT_SYNC.md

```diff
@@ -84,12 +82,18 @@ events.
 
 - `beforeUpdateCallback`
 a filter function which can be applied on a generated list of update actions. It allows the user to intercept product 
-update and modify (add/remove) update actions just before they are send to CTP API.
+updates and modify (add/remove) update actions just before they are send to CTP API.
```

Comment:
sent to .. would be nicer.

Reply:
I think you already fixed it in your prs. But doesn't hurt to change it here and there 😁 9809586